docs/non-normative/rpc-migration.md

```diff
@@ -0,0 +1,149 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: RPC migration
+--->
+
+# RPC semantic convention stability migration guide
+
+> [!WARNING]
+> This document is a work in progress as the RPC semantic conventions
+> have not been marked stable yet and changes are still being made.
+
+Due to the significant number of modifications and the extensive user base
+affected by them, existing RPC instrumentations published by
+OpenTelemetry are required to implement a migration plan that will assist users in
+transitioning to the stable RPC semantic conventions.
+
+Specifically, when existing RPC instrumentations published by OpenTelemetry are
+updated to the stable RPC semantic conventions, they:
+
+- SHOULD NOT change the version of the RPC conventions that they emit by
+  default in their existing major version. Conventions include (but are not
+  limited to) attributes, metric and span names, and unit of measure.
+- SHOULD introduce an environment variable `OTEL_SEMCONV_STABILITY_OPT_IN` in
+  their existing major version, which accepts:
+  - `rpc` - emit the stable RPC conventions, and stop emitting
+    the old RPC conventions that the instrumentation emitted previously.
+  - `rpc/dup` - emit both the old and the stable RPC conventions,
+    allowing for a phased rollout of the stable semantic conventions.
+  - The default behavior (in the absence of one of these values) is to continue
+    emitting whatever version of the old RPC conventions the
+    instrumentation was emitting previously.
+- Need to maintain (security patching at a minimum) their existing major version
+  for at least six months after it starts emitting both sets of conventions.
+- May drop the environment variable in their next major version and emit only
+  the stable RPC conventions.
+
+> [!NOTE]
+> `OTEL_SEMCONV_STABILITY_OPT_IN` is only intended to be used when migrating
+> from an experimental semantic convention to its initial stable version.
+
+## Summary of changes
+
+This section summarizes the changes made to the RPC semantic conventions
+from
+[v1.37.0](https://github.com/open-telemetry/semantic-conventions/blob/v1.37.0/docs/rpc/README.md).
+
+### RPC span attributes
+
+<!-- prettier-ignore-start -->
+| Change                                                         | PR                                                                        | Comments                                                     |
+| -------------------------------------------------------------- | ------------------------------------------------------------------------- | ------------------------------------------------------------ |
+| `rpc.system` &rarr; `rpc.system.name`                          | [#3176](https://github.com/open-telemetry/semantic-conventions/pull/3176) | See [below](#rpcsystemname-values) for changes to the values |
```

Comment:
not sure what folks think about adding the PR link here, I was planning to take it out before merging (it's really useful for reviewing), but maybe it's also useful to keep in..